get/get.go

```diff
@@ -28,7 +28,7 @@ type Cmd struct {
 	Clusters            clustersCmd           `cmd:"" group:"infrastructure.nine.ch" aliases:"cluster,vcluster" help:"Get Kubernetes Clusters."`
 	APIServiceAccounts  apiServiceAccountsCmd `cmd:"" group:"iam.nine.ch" name:"apiserviceaccounts" aliases:"asa" help:"Get API Service Accounts."`
 	Projects            projectCmd            `cmd:"" group:"management.nine.ch" name:"projects" aliases:"proj" help:"Get Projects."`
-	Applications        applicationsCmd       `cmd:"" group:"deplo.io" name:"applications" aliases:"app,apps" help:"Get deplo.io Applications."`
+	Applications        applicationsCmd       `cmd:"" group:"deplo.io" name:"applications" aliases:"app,apps,application" help:"Get deplo.io Applications."`
```

Comment:
```suggestion
	Applications        applicationsCmd       `cmd:"" group:"deplo.io" name:"applications" aliases:"app,apps,application,applications" help:"Get deplo.io Applications."`
```

Reply:
applications is the name of the cmd, so it's the default